terminal.go

```diff
@@ -493,13 +501,33 @@ func (g *LightningTerminal) start() error {
 			}},
 		}
 
+		var auxComponents lnd.AuxComponents
+		switch g.cfg.TaprootAssetsMode {
+		case ModeRemote, ModeDisable:
+			log.Warnf("Taproot Assets daemon is either disabled " +
+				"or running in remote mode. Taproot Asset " +
+				"channel functionality will NOT be " +
+				"available. To enable, set Taproot Assets " +
+				"mode to 'integrated' in the config file.")
+
+		case ModeIntegrated:
+			components, err := g.buildAuxComponents()
```

Comment:
yeah another point for hiding this behind the `SubServer` interface (but again - i'll do that)